content/zh-cn/docs/concepts/services-networking/service.md

```diff
@@ -804,6 +804,8 @@ Kubernetes `ServiceTypes` 允许指定你所需要的 Service 类型。
 * `ClusterIP`: Exposes the Service on a cluster-internal IP. Choosing this value
   makes the Service only reachable from within the cluster. This is the
   default that is used if you don't explicitly specify a `type` for a Service.
+  You can expose the service to the public with an [Ingress](docs/reference/kubernetes-api/service-resources/ingress-v1/) or the
```

Comment:
The English upstream is incorrect. But we can keep it that way until the upstream has a fix.